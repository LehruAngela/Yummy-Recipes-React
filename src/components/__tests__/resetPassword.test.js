import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import ResetPassword from '../resetPassword';

describe('ResetPassword', () => {
    const wrapper = shallow(<ResetPassword />);
    const preventDefault = jest.fn();

    it('renders properly', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
      }); 

    it('handles ResetPassword', () => {
        expect(wrapper.instance().handleResetPassword({ preventDefault }));
    });

 
});