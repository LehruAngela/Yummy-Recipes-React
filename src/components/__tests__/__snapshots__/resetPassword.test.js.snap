// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPassword renders properly 1`] = `
<div
  className="form-margin"
>
  <h2>
    Reset Password
  </h2>
  <br />
  <form
    onSubmit={[Function]}
  >
    <div
      className="form-group"
    >
      <h4>
        <label
          htmlFor="email"
        >
          Email:
        </label>
        <br />
      </h4>
      <input
        name="email"
        onChange={[Function]}
        type="email"
        value=""
      />
    </div>
    <div
      className="form-group"
    >
      <h4>
        <label
          htmlFor="password"
        >
          New Password:
        </label>
        <br />
      </h4>
      <input
        name="new_password"
        onChange={[Function]}
        type="password"
        value=""
      />
    </div>
    <div
      className="form-group"
    >
      <h4>
        <label
          htmlFor="confirmpassword"
        >
          Confirm New Password:
        </label>
        <br />
      </h4>
      <input
        name="confirm_new_password"
        onChange={[Function]}
        type="password"
        value=""
      />
    </div>
    <div>
      <button
        className="btn btn-outline-warning"
        type="submit"
      >
        Reset Password
      </button>
    </div>
    <br />
    <br />
  </form>
  <footer>
    <p
      class="copyright text-muted small footer"
    >
      Copyright Â© Gela 2018.
    </p>
  </footer>
</div>
`;
